src/main/java/com/palantir/gradle/versions/VersionsLockPlugin.java

```diff
@@ -103,7 +103,7 @@
 
 public class VersionsLockPlugin implements Plugin<Project> {
     private static final Logger log = Logging.getLogger(VersionsLockPlugin.class);
-    static final GradleVersion MINIMUM_GRADLE_VERSION = GradleVersion.version("5.3");
+    static final GradleVersion MINIMUM_GRADLE_VERSION = GradleVersion.version("6.1");
```

Comment:
This is imposed by `nebula-dependency-recommender:10.0.0`.

Reply:
We only need nebula-dependency-recommender for tests, right? In theory, we could replace it with our own gradle code that does the same stuff.

That said, I think it's fine to kill off Gradle 5 support now that 7 is out (probably want to major rev it).

Reply:
Yes, it's just for tests so not strictly raising the minimum gradle version for the actual plugin.